### ISSUE TITLE
Wrong IP address

### ISSUE THREAD

**Issue by JacVid** [kazcangi/homebridge-pioneer-avr#7]
Hello,

thanks for your plugin, it seems to be very intersting. But I have an issue regarding the IP adress config.

No matter the IP address I can put in the Config.json file, I've always got the following error

```
[5/18/2019, 4:35:34 PM] [Pioneer AVR] Received an error while communicating Error: connect ECONNREFUSED 127.0.0.1:23
[5/18/2019, 4:35:34 PM] Error: getaddrinfo ENOTFOUND undefined undefined:80
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:56:26)
    at Function.module.exports.loopWhile (/homebridge/node_modules/deasync/index.js:71:23)
    at Function.sleep (/homebridge/node_modules/deasync/index.js:41:18)
    at pioneerAvrAccessory.getServices (/homebridge/node_modules/homebridge-pioneer-avr/index.js:317:28)
    at Server._createAccessory (/usr/local/lib/node_modules/homebridge/lib/server.js:427:36)
    at Server._loadAccessories (/usr/local/lib/node_modules/homebridge/lib/server.js:309:26)
    at Server.run (/usr/local/lib/node_modules/homebridge/lib/server.js:91:38)
    at module.exports (/usr/local/lib/node_modules/homebridge/lib/cli.js:59:10)
    at Object.<anonymous> (/usr/local/lib/node_modules/homebridge/bin/homebridge:17:22)
    at Module._compile (internal/modules/cjs/loader.js:701:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:712:10)
    at Module.load (internal/modules/cjs/loader.js:600:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:539:12)
    at Function.Module._load (internal/modules/cjs/loader.js:531:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:754:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)
```

=> why does your plugin keep this 127.0.0.1 IP address ?

Thank you.

Jack

**Comment by kazcangi** [comment]
Hello,

It's a mistake  in the README file. The parameter ip must me named "host". 

Thanks for your feedback

Seb
```
"accessories": [
	{
        "accessory": "pioneerAvrAccessory",
        "model": "VSX-922",
        "name": "My Pioneer AVR",
        "description": "AV Receiver",
        "host": "192.168.178.99",
        "port": 23
	}
]
```

**Comment by kazcangi** [comment]
Fixed in 0.7.2
